package-lock.json

```diff
@@ -7393,9 +7393,8 @@
     },
     "node_modules/crypto-js": {
       "version": "4.2.0",
-      "resolved": "https://nexus.ci.diabeloop.eu/repository/npm-diabeloop/crypto-js/-/crypto-js-4.2.0.tgz",
-      "integrity": "sha512-KALDyEYgpY+Rlob/iriUtjV6d5Eq+Y191A5g4UqLAi8CyGP9N1+FdVbkc1SxKc2r4YAYqG8JzO2KGL+AizD70Q==",
-      "license": "MIT"
+      "resolved": "https://registry.npmjs.org/crypto-js/-/crypto-js-4.2.0.tgz",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Was this change wanted? Or is it an issue following a merge conflict? I find it weird to be retrieving the package directly from the NPM registry instead of Nexus